app/index.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { render } from 'react-dom';
 import { Provider } from 'react-redux';
 import { Router } from 'react-router';
+import { hashHistory } from 'react-router';
```

Comment:
Just curious, why hashHistory?
https://github.com/rackt/react-router/blob/latest/docs/guides/basics/Histories.md#should-i-use-createhashhistory
> Hash history is the default because it works without any setup on your server, and works in all evergreen browsers and IE8+. But, we don't recommend using it in production, every web app should aspire to use createBrowserHistory.